### PR TITLE
Add Footer layout with style

### PR DESCRIPTION
**Pull request by adrianMRJ**
# Description
What did you implemented/Why did you implemented this:

 - What?: W've added the footer layout and fix some issues caused of the modeling of the view.
 - Why?: These changes finish the landing view.

## Screenshots
- Mobile
image
- Desktop
image

### CODE REVIEW

**Review by JuanequeX — APPROVED** [review]
LGTM (Y)

**Review by fabiannabi — DISMISSED** [review]
LGTM

File: app/assets/images/icons/facebook.svg:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (jclopezdev) [inline app/assets/images/icons/facebook.svg:Zone.Identifier:1]: @adrianMRJ  Sorry, what do you need these files for?

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -1,22 +1,22 @@
-<div class ="main">
-  <div class="container">
-    <%= render_svg 'Mobile-Logo' %>
-    <h1 class="header">Log in</h1>
+<div class ="login">
```
Comment (jclopezdev) [inline app/views/devise/sessions/new.html.erb:1]: If this PR is only to add the Footer, you will need to remove these login / register changes, so your PR will look cleaner, what do you think?

File: stylesheets/views/_register.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    }
+  
+    &__header{
+      font-weight: 700;
+      margin-top: 1.5rem;
+    }
+  
+  }
+
```
Comment (jclopezdev) [inline stylesheets/views/_register.scss:33]: Please fix the indentation here and add only the blank line at the end of the file

File: app/assets/images/icons/instagram.svg:Zone.Identifier [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/
```
Comment (YilKanda) [inline app/assets/images/icons/instagram.svg:Zone.Identifier:3]: As @jclopezdev mentions, do you need this file?

File: app/assets/images/icons/logo-desktop.svg:Zone.Identifier [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/
```
Comment (YilKanda) [inline app/assets/images/icons/logo-desktop.svg:Zone.Identifier:3]: As @jclopezdev mentions, do you need this file?

File: app/assets/images/icons/logo-mobile.svg:Zone.Identifier [hunk 6]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/
```
Comment (YilKanda) [inline app/assets/images/icons/logo-mobile.svg:Zone.Identifier:3]: As @jclopezdev mentions, do you need this file?

File: app/assets/images/icons/twitter.svg:Zone.Identifier [hunk 7]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/
```
Comment (YilKanda) [inline app/assets/images/icons/twitter.svg:Zone.Identifier:3]: As @jclopezdev mentions, do you need this file?

File: stylesheets/base/_form.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
+.form {
+
```
Comment (YilKanda) [inline stylesheets/base/_form.scss:2]: Eliminate extra space

File: stylesheets/base/_form.scss [hunk 9]
```diff
@@ -0,0 +1,55 @@
+.form {
+    
+
+  @media screen and (min-width: $breakpoint-tablet) {
+    font-size: 1.5rem;
+  }
+
+  &__label {
+    color: #464343;
```
Comment (YilKanda) [inline stylesheets/base/_form.scss:9]: I notice that another file adds sass variables for color, but in this one you don't use them.

File: stylesheets/base/_form.scss [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+
+  &__link {
+    font-weight: 700;
+    text-decoration: none;
+  }
+
+  &__input {
+    background-color: #EBEAEA;
```
Comment (YilKanda) [inline stylesheets/base/_form.scss:24]: I notice that another file adds sass variables for color, but in this one you don't use them.

File: stylesheets/base/_form.scss [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+    border-radius: .2rem;
+    border: solid .05rem #D9D9D9;
```
Comment (YilKanda) [inline stylesheets/base/_form.scss:26]: I notice that another file adds sass variables for color, but in this one you don't use them.

File: stylesheets/base/_form.scss [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+    padding: .3rem;
+    @media screen and (min-width: $breakpoint-tablet) {
+      margin-left: 1rem;
+      margin-right: 1rem;
+    }
+  }
+
+  &__button {
+    background-color: #464343;
```
Comment (YilKanda) [inline stylesheets/base/_form.scss:35]: I notice that another file adds sass variables for color, but in this one you don't use them.

File: stylesheets/views/_register.scss [hunk 13]
```diff
@@ -0,0 +1,33 @@
+.register {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: 85vh;
+  
+    @media screen and (min-width: $breakpoint-tablet) {
+      background-color: #444444;
```
Comment (YilKanda) [inline stylesheets/views/_register.scss:9]: I notice that another file adds sass variables for color, but in this one you don't use them.

File: stylesheets/views/_register.scss [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  
+    &__container {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+  
+      @media screen and (min-width: $breakpoint-tablet) {
+        background-color: #F6F6F6;
```
Comment (YilKanda) [inline stylesheets/views/_register.scss:20]: I notice that another file adds sass variables for color, but in this one you don't use them.

**Review by YilKanda — COMMENTED** [review]
I agree with what @jclopezdev mentions, likewise, I also make some observations regarding the use of colors, remember to use a simple range of colors, so it is advisable to have a sass file where you call the colors you are using throughout your project in the form of variables.

Example:

```
$colors : (
  primary: #008080,
  secondary: #1877f2,
  disabled: gray,
);

@each $color_name,$color_value in $colors{
  .#{$color-name}-background{
    background-color: $color_value;
  }
  .#{$color-name}-text{
    color: $color_value;
  }
}
```

File: app/assets/images/icons/facebook.svg:Zone.Identifier (hunk 1 above)
Comment (adrianMRJ) [reply app/assets/images/icons/facebook.svg:Zone.Identifier:1]: They are auto generated, we delete them but it reapears

File: app/views/devise/sessions/new.html.erb (hunk 2 above)
Comment (adrianMRJ) [reply app/views/devise/sessions/new.html.erb:1]: We added those changes as hotfix, we didn't think it should need its own PR

File: stylesheets/views/_register.scss (hunk 3 above)
Comment (adrianMRJ) [reply stylesheets/views/_register.scss:33]: Done

File: app/assets/images/icons/instagram.svg:Zone.Identifier (hunk 4 above)
Comment (adrianMRJ) [reply app/assets/images/icons/instagram.svg:Zone.Identifier:3]: Its a autogenerated file, if we delete it, it reapears

File: app/assets/images/icons/logo-desktop.svg:Zone.Identifier (hunk 5 above)
Comment (adrianMRJ) [reply app/assets/images/icons/logo-desktop.svg:Zone.Identifier:3]: Its a autogenerated file, if we delete it, it reapears

File: app/assets/images/icons/logo-mobile.svg:Zone.Identifier (hunk 6 above)
Comment (adrianMRJ) [reply app/assets/images/icons/logo-mobile.svg:Zone.Identifier:3]: Its a autogenerated file, if we delete it, it reapears

File: app/assets/images/icons/twitter.svg:Zone.Identifier (hunk 7 above)
Comment (adrianMRJ) [reply app/assets/images/icons/twitter.svg:Zone.Identifier:3]: Its a autogenerated file, if we delete it, it reapears

File: stylesheets/base/_form.scss (hunk 9 above)
Comment (adrianMRJ) [reply stylesheets/base/_form.scss:9]: Its a autogenerated file, if we delete it, it reapears

File: stylesheets/base/_form.scss (hunk 10 above)
Comment (adrianMRJ) [reply stylesheets/base/_form.scss:24]: Its a autogenerated file, if we delete it, it reapears

File: stylesheets/base/_form.scss (hunk 11 above)
Comment (adrianMRJ) [reply stylesheets/base/_form.scss:26]: Done

File: stylesheets/base/_form.scss (hunk 8 above)
Comment (adrianMRJ) [reply stylesheets/base/_form.scss:2]: Done

File: stylesheets/base/_form.scss (hunk 12 above)
Comment (adrianMRJ) [reply stylesheets/base/_form.scss:35]: Done

File: app/assets/images/icons/logo-desktop.svg:Zone.Identifier (hunk 5 above)
Comment (jclopezdev) [reply app/assets/images/icons/logo-desktop.svg:Zone.Identifier:3]: But this shouldn't be marked as resolved just like that, what have you tried to solve it? These files must not be in the repository

**Review by jclopezdev — CHANGES_REQUESTED** [review]
Please remove the `Zone.Identifier` files

File: stylesheets/views/_register.scss (hunk 13 above)
Comment (adrianMRJ) [reply stylesheets/views/_register.scss:9]: Done

File: stylesheets/views/_register.scss (hunk 14 above)
Comment (adrianMRJ) [reply stylesheets/views/_register.scss:20]: Done

File: app/views/devise/sessions/new.html.erb (hunk 2 above)
Comment (jclopezdev) [reply app/views/devise/sessions/new.html.erb:1]: yes, you need to create a hotfix PR for that, and also, don't mark these conversations if are not really resolved

File: app/assets/images/icons/logo-desktop.svg:Zone.Identifier (hunk 5 above)
Comment (adrianMRJ) [reply app/assets/images/icons/logo-desktop.svg:Zone.Identifier]: I've deleted the files, they are no longer in the repository

Comment (adrianMRJ) [reply app/assets/images/icons/logo-desktop.svg:Zone.Identifier]: Done

**Pull request closed without merging**